lnrpc/walletrpc/walletkit.proto

```diff
@@ -389,6 +390,18 @@ message Account {
     // Whether the wallet stores private keys for the account.
     bool watch_only = 8;
 }
+
+message AddressBalance {
+    string address = 1;
+    int64 balance = 2;
+}
+
+message AddressesResponseType {
+    string name = 1;
```

Comment:
Same here and in the `ListAddressesRequest`, we should probably call this `account_name`.